### ISSUE TITLE
Support for Spot Instances

### ISSUE THREAD

**Issue by oelesinsc24** [aws-samples/aws-sagemaker-build#27]
@JohnCalhoun, with support training jobs on spot instances released yesterday (https://aws.amazon.com/blogs/aws/managed-spot-training-save-up-to-90-on-your-amazon-sagemaker-training-jobs/. Will this feature be added to SageMaker Build?

Thanks in advance

**Comment by JohnCalhoun** [comment]
yes, I will work on getting this into sagebuild. will be just an additional config parameter to turn on spot training

**Comment by oelesinsc24** [comment]
Any updates on this?

**Comment by JohnCalhoun** [comment]
just put in a pull request for this feature
